content/pages/meetup/2023/0719-nycu.rst

```diff
@@ -0,0 +1,70 @@
+========================================
+Meetup 2023 July 19th in NYCU
+========================================
+
+:date: 2023-07-15 16:06
+:url: meetup/2023/0719-nycu
+:save_as: meetup/2023/0719-nycu.html
+
+Date
+-----
+
+
+* Date: July 19th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:30 Free discussion
+* 19:30-21:30 Subject (discussion): Preparations for August scisprint  
+
+|
+
+About Meetup
+------------
+
+Meetup is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+We would like to provide a supportive and friendly environment for all 
+attendees to support more developers to join in the open-source communities. 
+
+To join the meetup, please bring your laptop. You are also very welcome to 
+bring your project. Please `contact us <#contact-us>`__ if you have any questions.
+
+Subjects
+------------------
+
+**Discussion: Preparations for August scisprint**
+
+Additionally, we will discuss the following items for preparing August scisprint. 
+
+    * Task arrangement for volunteers
+    * Promotion plan for August scisprint
+    * Preparations for career conversation
+
+|
+
+Venue
+-----
+`國立陽明交通大學 工程三館 3 樓 329 室 (Room 329, Engineering Building 3, NYCU) <https://goo.gl/maps/TgDYwohB3CBmQgww9>`__.
+
+.. raw:: html
+
+  <div style="overflow:hidden; padding-bottom:56.25%; position:relative; height:0;">
+    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d905.5596639949631!2d120.99673777209487!3d24.787280157478236!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3468360f96adabd7%3A0xedfd1ba0fa6c6bf7!2z5ZyL56uL6Zm95piO5Lqk6YCa5aSn5a24IOW3peeoi-S4iemkqA!5e0!3m2!1szh-TW!2stw!4v1678519228058!5m2!1szh-TW!2stw" 
+      style="left:0; top:0; height:100%; width:100%; position:absolute; border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade">
+    </iframe>
+  </div>
+
+Contact us
+----------
+
+* Discord: https://discord.gg/G3cccnGfMe
```

Comment:
We have a long-existing invite link https://discord.gg/6MAkFrD .  I think there does not need to be a new one.

Reply:
I see it is updated.  Thanks.  And please add a comment here to notify your change next time.